### DISCUSSION TITLE
Does AsciidocFX fully support EPUB 3.3?

### DISCUSSION THREAD

**Discussion by Korb** [asciidocfx/AsciidocFX#664] [Q&A]
`README.adoc` only says the following:

> AsciidocFX can converts Asciidoc documents into HTML, PDF, **EPUB3** and DOCBOOK5.

[EPUB 3.3 W3C Recommendation. 17 October 2024](https://www.w3.org/TR/epub-33/)

**Comment by rahmanusta** [comment]
That’s pretty new, I don’t think so. It is based on https://github.com/asciidoctor/asciidoctor-epub3, you may ask about it to them. What exact feature in epub3.3 do you need?

**Reply by Korb** [reply]
The changes mainly concern the adaptation of content for people with disabilities. I cannot answer which disabilities are more important for checking the compatibility of a book with them. I would prefer a quantitative comparison of the added features of the EPUB 3.3 standard in different editors.
